src/libstd/sys/unix/stack_overflow.rs

```diff
@@ -60,7 +60,11 @@ mod imp {
     // This is initialized in init() and only read from after
     static mut PAGE_SIZE: usize = 0;
 
-    #[cfg(any(target_os = "linux", target_os = "android"))]
+    #[cfg(any(target_os = "linux",
+              target_os = "android",
+              target_os = "bitrig",
```

Comment:
It looks like bitrig already defines the `si_addr` field, how come this was necessary?

```rust
pub struct siginfo_t {
    pub si_signo: ::c_int,
    pub si_code: ::c_int,
    pub si_errno: ::c_int,
    pub si_addr: *mut ::c_void
}
```